frame/staking-rewards/src/lib.rs

```diff
@@ -590,14 +601,21 @@ pub mod pallet {
 			// Transfer the reward locally.
 			let protocol_account = Self::account_id(asset);
 			T::Assets::transfer(*reward_asset, from, &protocol_account, amount, keep_alive)?;
-			EpochRewards::<T>::try_mutate(
-				(Self::current_epoch(), *asset),
-				reward_asset,
-				|collected_amount| -> DispatchResult {
-					*collected_amount = collected_amount.safe_add(&amount)?;
-					Ok(())
-				},
-			)?;
+			let f = |rewards: &mut RewardsOf<T>| -> DispatchResult {
```

Comment:
I generally find one letter variable names hard to read and understand what they do :)
```suggestion
			let reward_collector = |rewards: &mut RewardsOf<T>| -> DispatchResult {
```